js/src/popover.js

```diff
@@ -110,12 +110,21 @@ class Popover extends Tooltip {
 
     // we use append for html objects to maintain js events
     this.setElementContent(SelectorEngine.findOne(Selector.TITLE, tip), this.getTitle())
+    // if the popover does not have a title we delete the .popover-header tag
+    if (!this.getTitle() && SelectorEngine.findOne(Selector.TITLE, tip)) {
```

Comment:
you should store the result of `SelectorEngine.findOne(Selector.TITLE, tip)`, it'll allow you to use it in your `if` case and it's better in terms of performances

Reply:
I changed my solution so this part was removed